src/Modules/CalcSetup.lua

```diff
@@ -54,6 +54,8 @@ function calcs.initModDB(env, modDB)
 	modDB:NewMod("TotemPlacementTime", "BASE", 0.6, "Base")
 	modDB:NewMod("BallistaPlacementTime", "BASE", 0.35, "Base")
 	modDB:NewMod("ActiveTotemLimit", "BASE", 1, "Base")
+	modDB:NewMod("ShockStacksMax", "BASE", 0, "Base")
+	modDB:NewMod("ScorchStacksMax", "BASE", 0, "Base")
```

Comment:
These should be 1 by defualt

Reply:
Couldn't make these 1, then I'd have to reduce the num by 1 when parsing the max stack mod

Reply:
Ah i see. Kinda worried someone may use those without realizing that but i get it.